dist.Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3-slim
```

Comment:
Should we specify the 3 version here? I guess it should be whatever highest version we claim compatibility with? I think maybe that's 3.9?

Reply:
update to `python:3.9-slim` :)